mantidproject/mantid
# Reorder export as batch keys to mirror sans table

**Description of work.**
Data in the ISIS SANS table can be exported as a batch file. This PR reorders how the data is written to a csv so that data terms in the batch file appear in the same order as in the table.

**To test:**
1. Interfaces -> SANS -> ISIS SANS
2. Type any data in any of first 7 columns of the table (numbers in the first 6, a name in "Output Name")  
3. Click `export table` (directly above the table) and save it
4. Open up the csv. Compare the order of the data labels with the order of the column titles in the ISIS SANS table. They should align. (The format of the batch file is key1 | value1 | key2 | value2 etc., so some columns will be empty if you didn't provide data in each table column)
5. Back in the ISIS SANS interface, highlight the row you populated and lcick the bin icon
6. Click `Load Batch File` in the upper right corner and select the csv file
7. This should populate the table with the row you created earlier

Fixes #25538 


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
